fix: minor typo in ads template

software-dov opened the pull request:
Closes #631

software-dov committed:
fix: minor typo in ads template

busunkim96 reviewed (approved):
Is this code not being exercised in the unit tests?

software-dov commented:
It looks like it isn't. The `showcase-alternative-templates-*` tasks should be stressing the ads templates, but local experimentation indicates that they're not actually using the correct template paths.

software-dov committed:
fix: CI runs alternative template tests

CircleCI machinery now invokes the alternative (Ads) templates for the
showcase_alternative_templates_* tests.

Includes numerous fixes and additions to the Ads grpc transport,
client class, and unit test templates.

The showcase system tests now selectively enable async tests via an
environment variable. The async client code has not yet been added to
the Ads templates, and the corresponding system tests have been
disabled for alternative templates.

software-dov commented:
Turns out the template path arg wasn't being passed through the noxfile :P
Fixing that has unloaded an entire yak's worth of ancillary changes. I'll update this PR with the result.

software-dov committed:
Satiate style check

software-dov committed:
More style check

software-dov committed:
Fix for 3.6 dependency stuff

software-dov committed:
Fix for showcase_mtls

software-dov committed:
Expose (sync) client transport property to async client

software-dov committed:
Tweak

software-dov committed:
Merge branch 'master' into ads-experiment

software-dov committed:
Don't depend on showcase-unit-alternative-templates-3.6

software-dov committed:
Merge branch 'ads-experiment' of https://github.com/software-dov/gapic-generator-python into ads-experiment
